### PR TITLE
Add the reliable-Keyspace

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: docs/snippets/examples/reliable-keyspace-notification.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+GearsBuilder() \
+.foreach(lambda x: execute('XADD', "notifications-stream", '*', *sum([[k,v] for k,v in x.items()],[]))) \
+.register(prefix="person:*", eventTypes=['hset', 'hmset'])
```
Comment (MeirShpilraien) [inline docs/snippets/examples/reliable-keyspace-notification.py:3]: What about the other hset commands? del? rename?

**Review by MeirShpilraien — COMMENTED** [review]
@gkorland why not creating a recipe that handles all the edge cases like rename/del maybe can handle more datatypes?

File: docs/snippets/examples/reliable-keyspace-notification.py (hunk 1 above)
Comment (gkorland) [reply docs/snippets/examples/reliable-keyspace-notification.py:3]: it's just an example, similar to using `prefix="person:*"`

**Comment by gkorland** [comment]
@MeirShpilraien I'll generate a full recipe too, it's just nice simple example

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**